backend/src/packages/chaiNNer_pytorch/__init__.py

```diff
@@ -4,11 +4,28 @@
 
 from api import GB, KB, MB, Dependency, add_package
 from gpu import nvidia_is_available
+from system import is_arm_mac
 
 python_version = sys.version_info
 
 
 def get_pytorch():
+    # 1.13.1 can take advantage of MPS
+    if is_arm_mac:
+        return [
+            Dependency(
+                display_name="PyTorch",
+                pypi_name="torch",
+                version="1.13.1",
+                size_estimate=140 * MB,
+            ),
```

Comment:
No special link this time?

Reply:
no, because its on pypi. only the cuda version needs the findlink